jpype/_jarray.py

```diff
@@ -104,6 +116,10 @@ def __iter__(self):
     def __reversed__(self):
         for elem in self[::-1]:
             yield elem
+    
+    def __contains__(self, item):
+        # "in" works without this but this should be more efficient
+        return _jpype.JClass("java.util.Arrays").asList(self).contains(item)
```

Comment:
Was this change documented.   It seems like a separate enhancement.  I would recommend that to avoid multiple lookups it may be best to add it to the ``_core.py`` typecache.

```
   _jpype._java__util_Array = _jpype.JClass("java.util.Array")
```

Though the type look up system is fairly fast and using "in" as part of loop is likely going to be slow anyway.

Reply:
May as well, there are several other places in jpype where Arrays is used where it could benefit such as the auto conversions.

I didn't document the change, there is no behavior change that this introduces. I think Python just automatically iterates over it and checks if there is no `__contains__` 

The reason I added it was so that it completes the Sequence interface.

Reply:
Well, surprise... I got `List` of the array and not a `List` of the contents. Not sure how I managed to goof that up but I had assumed there was already test coverage since `in` already worked without a problem. In order to get the correct `List` I'd need to do `_jpype.JClass("java.util.Arrays").asList(*self)` which defeats the purpose so I'm going to drop this change.

Reply:
why not use a binary search here? https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Arrays.html
So you can avoid creating a temporary list.